apps/astarte_pairing/lib/astarte_pairing/fdo/owner_onboarding/ov_next_entry.ex

```diff
@@ -18,24 +18,35 @@
 
 defmodule Astarte.Pairing.FDO.OwnerOnboarding.OVNextEntry do
   @moduledoc """
-  TO2.OVNextEntry (Msg 63): The Owner sends a specific Ownership Voucher entry.
-
+  TO2.OVNextEntry (Type 63).
   From Owner Onboarding Service to Device ROE.
+
+  This message delivers a single Ownership Voucher entry to the Device.
+  It is the response to TO2.GetOVNextEntry (Msg 62).
+
+  The OVEntry is essentially a link in the certificate chain that allows the Device
+  to verify that the current Owner is legitimate and traces back to the manufacturer.
+
+  Spec Reference: 5.5.5 TO2.OVNextEntry, Type 63
   """
   use TypedStruct
+  alias Astarte.Pairing.FDO.OwnerOnboarding.OVNextEntry
 
   typedstruct enforce: true do
     @typedoc "Structure for TO2.OVNextEntry message."
 
-    # The index of the entry being sent. Must match the request from Msg 62.
+    # OVEntryNum
+    # The index of the entry being sent. Must match the request received in Msg 62.
     field :entry_num, non_neg_integer()
```

Comment:
Let's make this uniform: we're using the FDO naming also for field names, this would be `:ov_entry_num`